common/src/commonMain/kotlin/io/xorum/codeforceswatcher/features/users/redux/requests/UsersRequests.kt

```diff
@@ -101,4 +101,11 @@ class UsersRequests {
 
         data class Failure(override val message: Message) : ToastAction
     }
+
+    class Destroy : Request() {
+
+        override suspend fun execute() {
+            // TODO DatabaseQueries.Users.deleteAll()
+        }
```

Comment:
Already implemented?

Reply:
Yes, but not sure that it will help now.